src/app/features/psbt-signer/components/psbt-request-details/components/psbt-inputs-and-outputs/components/psbt-input-list/components/psbt-input-item.tsx

```diff
@@ -0,0 +1,26 @@
+import { truncateMiddle } from '@stacks/ui-utils';
```

Comment:
Just on folder structure

<img width="314" alt="image" src="https://github.com/hirosystems/wallet/assets/1618764/41589710-1904-46ba-83e6-fae905b61c0f">

Is there a logical way to flatten this out? Odd to have `components/../components/../components`.

Reply:
I think that's actually good as then it's grouping the components for that area there. 

Sometimes I would promote something big like `psbt-inputs-and-outputs` to being a `feature` of `psbt-request-details` (like how we use `pages` )

Reply:
I do like keeping the `components` folder as I have it. I think it makes the folder structure easier to read and seems more organized. Good point, @pete-watters, about the feature. I'll think on that one to see if will work here.